localstack/services/ses/ses_starter.py

```diff
@@ -25,6 +27,8 @@ def email_responses_send_raw_email(self):
 
         raw_data = to_str(base64.b64decode(self.querystring.get('RawMessage.Data')[0]))
 
+        LOGGER.error('Raw email:\n%s' % raw_data)
```

Comment:
Error log level seem a bit too verbose here - can we use `LOGGER.info` instead?

Reply:
Doh... I wanted debug.

Reply:
fixed